apps/ui/src/views/Space/Proposals.vue

```diff
@@ -95,26 +95,26 @@ watchEffect(() => setTitle(`Proposals - ${props.space.name}`));
             {
               key: 'pending',
               label: 'Pending',
-              component: ProposalStatusIcon,
+              component: ProposalIconStatus,
               componentProps: { ...selectIconBaseProps, state: 'pending' }
             },
             {
               key: 'active',
               label: 'Active',
-              component: ProposalStatusIcon,
+              component: ProposalIconStatus,
               componentProps: { ...selectIconBaseProps, state: 'active' }
             },
             {
               key: 'closed',
               label: 'Closed',
-              component: ProposalStatusIcon,
+              component: ProposalIconStatus,
               componentProps: { ...selectIconBaseProps, state: 'passed' }
             }
           ]"
         />
       </div>
       <div v-if="!network.readOnly" class="flex flex-row p-4 space-x-2">
-        <VotingPowerIndicator
+        <SpaceIndicatorVotingPower
```

Comment:
I wonder should it be prefixed with `Space`? It's used in `Space` view, but also in `Proposal` view.

Reply:
Yeah probably not, changed it